components/Common/ShellBox/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { useRef } from 'react';
+import { FormattedMessage } from 'react-intl';
+import styles from './index.module.scss';
+import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
+import type { FC, PropsWithChildren, MouseEvent, ReactNode } from 'react';
+
+type ShellBoxProps = {
+  children: string | ReactNode;
```

Comment:
We shouldn't allow plain strings. Just ReactNode's (which in theory supports strings). But I guess this is fine...